core-bundle/src/Resources/contao/templates/rss/rss_items_only.html5

```diff
@@ -3,7 +3,7 @@
 <?php $this->block('content'); ?>
 
   <?php foreach ($this->items as $item): ?>
-    <div class="rss_items_only<?= $item['class'] ?>">
+    <div class="rss_items_only<?= $item['class'] ?? '' ?>">
```

Comment:
the `rss_...` templates are not used anymore, right?

Reply:
Yes, these are obsolete!

Reply:
Fixed in 0a29f8a8d